### ISSUE TITLE
master and tools branches...

### ISSUE THREAD

**Issue by Sopel97** [nodchip/Stockfish#325]
Since we now have the pytorch trainer and the architecture is becoming more complicated and less CPU friendly for training this trainer is less useful. It started causing issues when trying to merge with official-stockfish and polluted the codebase with costly (in terms of performance and maintanance) stuff that's only related to training. That's why I made a tools branch which is everything but with the training code removed. I now try to keep it up to date with official-stockfish/master. The question right now is, what should we do with the master branch? It's hard to update, and not very useful anymore. I'm currently considering moving the current master branch to some "legacy_master" branch, and put the tools branch as the new master. I want to spark some discussion, I want to know what people are using from this repository, and what you would like to see, what is the most useful for you, etc.

**Comment by fsmosca** [comment]
What are tools branches?

I have been using this master for generation of training pos and learning. So far it is usually better than pytorch trainer. I have mixed results with pytorch sometimes it can equalized the sf tainer. These comparisons are based on the same data but trained on sf and pytorch.

**Comment by Sopel97** [comment]
https://github.com/nodchip/Stockfish/tree/tools, which is closer to official-stockfish/master than nodchip/master but doesn't contain the learning part

**Comment by fsmosca** [comment]
This master is the origin of Sf nnue. It contains both training pos generation and learning, I like this repo to be as it is.

Stockfish master explores everything in order to get the highest performance possible. In the future new ideas/technology will be discovered. Following it is not easy.

I will be fine if you create a new repo with your changes on it.

**Comment by Sopel97** [comment]
I tend to agree. One other option could be to have the tools branch be in official-stockfish.
